File: apps/sepuling-java/src/test/java/smecalculus/rolevod/storage/SepulkaDaoMyBatisH2IT.java
package smecalculus.rolevod.storage;

import org.springframework.test.context.ContextConfiguration;
import smecalculus.rolevod.construction.SepulkaDaoBeans;
import smecalculus.rolevod.construction.StoragePropsBeans;

@ContextConfiguration(classes = {StoragePropsBeans.MyBatisH2.class, SepulkaDaoBeans.MyBatis.class})
public class SepulkaDaoMyBatisH2IT extends SepulkaDaoIT {}

File: apps/sepuling-kotlin/src/test/kotlin/smecalculus/rolevod/storage/SepulkaDaoMyBatisH2IT.kt
package smecalculus.rolevod.storage

import org.springframework.test.context.ContextConfiguration
import smecalculus.rolevod.construction.SepulkaDaoBeans
import smecalculus.rolevod.construction.StoragePropsBeans

@ContextConfiguration(classes = [StoragePropsBeans.MyBatisH2::class, SepulkaDaoBeans.MyBatis::class])
class SepulkaDaoMyBatisH2IT : SepulkaDaoIT()
